package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@citrineos/workspace",
```

Comment:
I wonder if we want to instead have this in the `server` and `swarm` directory.

Reply:
In theory, since we can set the paths it should be possible. Most examples I saw have a parent package.json and a `packages` folder that contains all of the modules and in the package.json they can just do ```"workspaces": [ "./packages/*]```. NPM then figures out all of the dependencies and creates a node_modules dir in the root which has all the shared dependencies and each module may, but does not need to, contain its own node_modules directory if they have any specific dependencies that arent used elsewhere. 

If we move this to Server, I would be concerned that we may run into some hick ups with the workspaces setup, but in theory should be possible. I would take the approach of just having `src/index.ts` to init the modules and eliminate the need for `server`. Swarm variation of server and various compositions can all be achieved by modifying env vars and docker files